ETL/Provision/provision_lambda.py

```diff
@@ -0,0 +1,71 @@
+"A script to read the url, and product data from the database"
+from os import environ as ENV
+import logging
+
+from dotenv import load_dotenv
+import psycopg2
+import psycopg2.extras
+from psycopg2.extensions import connection, cursor
+
+
+def get_connection(config) -> connection:
+    "Establishes a connection with the database"
+    return psycopg2.connect(
+        user=config["DB_USER"],
+        password=config["DB_PASSWORD"],
+        host=config["DB_HOST"],
+        port=config["DB_PORT"],
+        database=config["DB_NAME"]
+    )
+
+
+def get_cursor(conn: connection) -> cursor:
+    "Returns a cursor for the database"
+    return conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
```

Comment:
It could be worth testing that a connection object is passed (This also provides you with the ability to test this function):
```suggestion
def get_cursor(conn: connection) -> cursor:
    "Returns a cursor for the database"
    if not isinstance(conn, connection):
        raise TypeError('A cursor can only be constructed from a Psycopg2 connection object')
    return conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
```

Reply:
done